lib/route_translator/translator/path/segment.rb

```diff
@@ -16,17 +16,13 @@ def fallback_options(str, locale)
           def translate_string(str, locale, scope)
             locale = locale.to_s.gsub('native_', '')
             handler = proc { |exception| exception }
```

Comment:
`handler` is there to make I18n return an error (`I18n::MissingTranslation`) instead of a string (`translation missing:...`). 
It is possible to match on this string, as it was before. It feels like an indirect matching on the error though.